### DISCUSSION TITLE
Is there any way to change xAxis to date

### DISCUSSION THREAD

**Discussion by Dongseung** [nhn/tui.chart#724] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/75294699/156484408-30327583-c54b-462c-b3ec-52c365498c58.png)

I have set the label of yAxis as categories, but I want to change xAxis to date as MM/DD. 
Is there any api or options to chage it?

I have set as :
`theme: {
    xAxis: {
    date: { format: 'MM/DD' },
  },`

However, it does not work...😂

I want to make a chart as below image.
![image](https://user-images.githubusercontent.com/75294699/156484694-ada1c0d0-f6ec-4ecb-8297-88646724fc31.png)

**Comment by jwlee1108** [comment, marked as answer]
@Dongseung 

Hello, Are you using Barchart's Range function?
I guess you can use `fomatter` [option](https://github.com/nhn/tui.chart/blob/main/docs/ko/common-axes.md#formatter).

```
const options = {
  xAxis: {
    label: {
      formatter: (value) => {
        // @todo something.
      },
    },
  },
}
```

**Reply by Dongseung** [reply]
thank you. problem solved
